README.md

```diff
@@ -38,6 +52,63 @@ for chunk in stream:
   print(chunk['message']['content'], end='', flush=True)
 ```
 
+## Custom client
+
+A custom client can be created with the following fields:
+
+- `host`: The Ollama host (default: `http://localhost:11434`)
+- `timeout`: The timeout for requests (default: `None`)
+- `follow_redirects`: Whether to follow redirects (default: `True`)
+- `headers`: Additional headers to send with requests (default: `{}`)
```

Comment:
The user is not restricted to these fields. Any extra kwargs are passed verbatim to `httpx.Client`. These are mentioned only because they're default values differ from `httpx`

Reply:
Okay - let me make that callout instead